src/app/home/home.page.scss

```diff
@@ -0,0 +1,35 @@
+#container {
+  text-align: center;
+
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+#container strong {
+  font-size: 20px;
+  line-height: 26px;
+}
+
+#container p {
+  font-size: 16px;
+  line-height: 22px;
+
+  color: #8c8c8c;
+
```

Comment:
```suggestion
```

Reply:
Thanks for all the help @FlyteWizard :) We didn't have time to format the syntax today, but hopefully, future edits will be all cleaned up. I'm gonna merge for tomorrow. Thanks again!